### PR TITLE
fix: fix the false comments at DistributionWrapper.sol

### PR DESCRIPTION
**Pull request by ethisscam**
The comments of setWithdrawAddress and delegate functions are wrong. I fix them with comments from Distribution.sol and Staking.sol

## Summary by CodeRabbit

- **New Features**
  - Users can now set a specific address to receive delegation rewards in the `DistributionWrapper` contract.
  - Enhanced the `delegate` function to allow users to delegate tokens directly to a validator.

- **Documentation**
  - Corrected spelling errors in the documentation comments for the `IDistributionModule` interface.

- **Refactor**
  - Updated function descriptions and comments for clarity in the `DistributionWrapper` contract.

### CODE REVIEW

**Comment by ethisscam** [comment]
it seems the name of Deploy.sol contract is false too.

File: contracts/src/testing/DistributionWrapper.sol [hunk 1]
```diff
@@ -60,8 +60,8 @@ contract DistributionWrapper {
     }
 
     /**
-     * @dev Withdraw the rewrads accumulated by the caller(msg.sender).
-     * @param _withdrawAddress The address of the delegator.
+     * @dev The caller (msg.sender) can set the address that will receive the deligation rewards.
```
Comment (itsdevbear) [inline contracts/src/testing/DistributionWrapper.sol:63]: delegation**

**Review by itsdevbear — CHANGES_REQUESTED** [review]
typo

**Comment by ethisscam** [comment]
wait a minute. I directly copy the comment from Distribution.sol so there must be typo there too. Will submit later.

**Review by itsdevbear — APPROVED** [review]

**Pull request merged**